### ISSUE TITLE
3/24 - R Shiny - STO 442 - 

### ISSUE THREAD

**Issue by ashiklom** [bulib/studyGroup#35]
[Shiny](http://shiny.rstudio.com/) is a tool for creating interactive, web-based visualizations using R...but that's basically all I know about it other than that it looks pretty cool.

If anybody knows enough about it to lead a tutorial, that would be pretty cool. Otherwise, it might be fun to do a "co-learning" session where a few of us get together and try to figure it out.

**Comment by istfer** [comment]
I'd be very interested in learning shiny too, co-learnin session or whatever, I'm in

**Comment by tomhohenstein** [comment]
We could do a co-working session on March 24. @istfer would you be interested in doing a quick, 5-minute intro to Shiny to start the session?

**Comment by istfer** [comment]
sure thing

**Comment by tomhohenstein** [comment]
@istfer looking forward to this week's workshop. If there is anything I can do to help logistically, let me know!

**Comment by tomhohenstein** [comment]
Also - any chance we can use that conference room again? Seemed like a good spot

**Comment by istfer** [comment]
@tomhohenstein thanks! I haven't done this before, I was thinking of bringing myself and my laptop to give a 5 min intro, what else I need to have arranged?

**Comment by tomhohenstein** [comment]
@istfer I think a little intro about R Shiny and maybe an example would be good (esp. if you have an example from your work). From there we could spend time hacking around with it. 

Thoughts?

**Comment by istfer** [comment]
@tomhohenstein OK, I'll try to come up with something. Is arranging the projector/classroom also on me or does someone else take care of it?

**Comment by tomhohenstein** [comment]
we normally use STO 442 but we used a conference room last week. If you have a space preference, we can try to reserve either. Also, it seems like either @ceholden or myself tend to grab a projector from earth and environment (and I'm happy to do that again this week).

**Comment by istfer** [comment]
Thanks! No, no preferences. Just wanted to make sure I'm not missing anything. See you on Thursday.

**Comment by tomhohenstein** [comment]
bad news - my daughter is not feeling well and I'm at home with her today. can anyone step in to make sure @istfer has a projector?

**Comment by ashiklom** [comment]
Sorry to hear that @tomhohenstein, hope she feels better! I have a class on the first floor till right around 5, so I'll swing by the main office and grab one on my way back upstairs.

**Comment by tomhohenstein** [comment]
Great and thank you -  how did the session go?

**Comment by ashiklom** [comment]
Session went well! @istfer gave a brief but effective introduction to Shiny that provided all of the information we needed to get started, and then we spent the next hour playing with it to see what we could come up with. It's a great tool that is surprisingly easy to use (at least, for basic apps) while also providing a lot of potential for more advanced functionality.

Here is the website of the amerifluxr package that @istfer showed as an example: http://khufkens.github.io/amerifluxr/

There are also great tutorials and documentation on the [Shiny homepage](http://shiny.rstudio.com/).

Attendance: 5
